remote-wallet/Cargo.toml

```diff
@@ -36,6 +36,10 @@ solana-signature = { workspace = true, features = ["std"] }
 solana-signer = { workspace = true }
 thiserror = { workspace = true }
 uriparse = { workspace = true }
+serde_json = { workspace = true }
+hex = { workspace = true }
+ur-registry = { git = "https://github.com/KeystoneHQ/keystone-sdk-rust.git", tag = "0.0.50", default-features = false, features = ["std"] }
+ur-parse-lib = { git = "https://github.com/KeystoneHQ/keystone-sdk-rust.git", tag = "0.0.50" }
```

Comment:
If we want to use these crates, they'll need to be published to crates.io, otherwise we can't publish solana-remote-wallet. Is there a plan to publish soon?